packages/daemon/src/generator/tvm.ts

```diff
@@ -0,0 +1,20 @@
+import { fork } from 'child_process';
+import { GenerateOptions } from '../services/interface';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+
+export async function generateTVM(dist: string, projPackage: any, opts: GenerateOptions): Promise<void> {
```

Comment:
If `projPackage` is a object, it should be `Record<string, any>`.
And this function should be tested.